### PR TITLE
Changed the directory structure for tools/sim_qurt

### PR DESCRIPTION
**Pull request by aankit-ca**
1. Moved tools/sim_qurt/sim_qurt.cpp to src/runtime/hexagon_remote/
2. Modified the src/runtime/hexagon_remote/Makefile to build libsim_qurt.a
3. Removed duplicate code from sim_remote.cpp
4. Link libsim_qurt.a with hexagon_sim_remote for definitions of qurt_hvx_lock
   and qurt_hvx_unlock functions
5. Deleted tools/sim_qurt directory from tree

### CODE REVIEW

**Comment by aankit-ca** [comment]
@dsharletg PTAL

File: src/runtime/hexagon_remote/Makefile [hunk 1]
```diff
@@ -152,6 +153,13 @@ bin/%/sim_host.o: sim_host.cpp sim_protocol.h
 	mkdir -p $(@D)
 	$(CXX-$*) -std=c++11 $(CCFLAGS-$*) -c sim_host.cpp -o $@
 
+bin/%/sim_qurt/sim_qurt.o: sim_qurt.cpp
+	mkdir -p $(@D)
+	$(CXX-$*) $(CCFLAGS-$*) -O3 -c sim_qurt.cpp -o $@
```
Comment (pranavb-ca) [inline src/runtime/hexagon_remote/Makefile]: minor: -O3 is part of COMMON_FLAGS which is part of CCFLAGS-v60

Comment (aankit-ca) [reply src/runtime/hexagon_remote/Makefile]: Corrected

File: src/runtime/hexagon_remote/Makefile [hunk 2]
```diff
@@ -78,7 +78,8 @@ hosts: \
 remotes: \
 	bin/v60/libhalide_hexagon_remote_skel.so \
 	bin/v60/signed_by_debug/libhalide_hexagon_remote_skel.so \
-	bin/v60/hexagon_sim_remote
+	bin/v60/hexagon_sim_remote \
+	bin/v60/sim_qurt/libsim_qurt.a
```
Comment (dsharletg) [inline src/runtime/hexagon_remote/Makefile]: I think this should probably not be in a subfolder, just in `bin/v60` directly

Comment (aankit-ca) [reply src/runtime/hexagon_remote/Makefile]: Moved it to bin/v60

**Pull request merged**